doc/psidk/src/run-infrastructure/administration.md

```diff
@@ -91,6 +94,28 @@ Each peer has the following fields:
 |-----------|--------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | `service` | String | The name of the verify service                                                                                                                                      |
 | `rawData` | String | A hex string containing private key information. The interpretation depends on the verify service. If `rawData` is not present, the server will generate a new key. |
+| `device`  | String | The id of the cryptographic device which will hold the new key. The device must be unlocked.                                                                        |
+
+`GET` on `/native/admin/keys/devices` lists available cryptographic devices that can store keys.
+
+| Field      | Type    | Description                                          |
+|------------|---------|------------------------------------------------------|
+| `name`     | String  | The human readable name of the device                |
+| `id`       | String  | An opaque string that uniquely identifies the device |
+| `unlocked` | Boolean | `true` if the device is currently unlocked           |
+
+`POST` to `/native/admin/keys/unlock` unlocks a cryptographic device.
+
+| Field    | Type   | Description                 |
+|----------|--------|-----------------------------|
+| `device` | String | The device id.              |
```

Comment:
It would be nice if the device could be unlocked with the device name/label

Reply:
This is machine interface, not a human interface. I expect the UI to list devices by name, but use the id for requests. The name has a higher chance of being non-unique.